src/App.jsx

```diff
@@ -1,104 +1,29 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId) || null;
+}
+
+function getСommentsByPostId(postId) {
```

Comment:
The function name `getСommentsByPostId` contains a Cyrillic 'С' instead of a Latin 'C'. This can lead to unexpected behavior. Please replace the Cyrillic 'С' with a Latin 'C' to correct the function name.